desc/compat.py

```diff
@@ -111,6 +111,46 @@ def flip_helicity(eq):
     return eq
 
 
+def flip_theta(eq):
+    """Change the gauge freedom of the poloidal angle of an Equilibrium.
```

Comment:
If we flip this and then re-solve or optimize will it mess with the existing gauge freedom constraint on lambda?

Reply:
No it should be fine. The `FixLambdaGauge` constraint enforces $\lambda(\theta=0,\zeta=0)=0$ and this will still be true. All this function does is change $\theta=0$ from the inboard to outboard side (or vice-versa).

Reply:
Unless that is what you meant? The two different boundary inputs are physically equivalent, but it might be possible that numerically they converge slightly differently during an equilibrium solve. Conceptually it's similar to positive vs negative Jacobian for the same physical equilibrium.

Reply:
the sign should be the same, so I think it's fine.